src/components/Button.tsx

```diff
@@ -0,0 +1,11 @@
+import Button from '@mui/material/Button';
+
+const BasicButtons = ({text}:any) => {
```

Comment:
If you want to add secondary button you should add props color there

Reply:
done

Thanks, for helping me solve this, I couldn't figure this out :)